### PR TITLE
Improve verifier messages for secondary locations; enable Jacoco html reports locally

### PR DESCRIPTION
**Pull request by petertrr**

### CODE REVIEW

File: build-logic/src/main/kotlin/org.sonarsource.iac.java-conventions.gradle.kts [hunk 1]
```diff
@@ -41,7 +41,7 @@ tasks.jacocoTestReport {
     reports {
         xml.required.set(true)
         csv.required.set(false)
-        html.required.set(false)
+        html.required.set(providers.environmentVariable("CI").map(String::toBoolean).orElse(true))
```
Comment (jonas-wielage-sonarsource) [inline build-logic/src/main/kotlin/org.sonarsource.iac.java-conventions.gradle.kts:44]: So basically locally its always false?

Comment (jonas-wielage-sonarsource) [inline build-logic/src/main/kotlin/org.sonarsource.iac.java-conventions.gradle.kts:44]: Shouldn't it be false when we're in ci?
`html.required.set(!providers.environmentVariable("CI").isPresent)`

Comment (petertrr) [reply build-logic/src/main/kotlin/org.sonarsource.iac.java-conventions.gradle.kts]: Yes, good catch, would be a silly mistake :sweat_smile:

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**